contracts/utils/Helpers.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4 <0.9.0;
+
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @dev Library of helpers for generating SQL statements from common parameters.
+ */
+library Helpers {
+    /**
+     * @notice Generates a properly formatted table name from a prefix and table id.
+     * @param prefix the user generated table prefix as a string.
+     * @param tableId the Tableland generated tableId as a uint256.
+     * @return name newly allocated string containing the table name.
+     *
+     * @dev requirements: block.chainid must refer to a supported chain.
+     */
+    function getNameFromId(string memory prefix, uint256 tableId)
+        public
+        view
+        returns (string memory)
+    {
+        return
+            string.concat(
```

Comment:
I believe this style of concatenating strings is only available in  solidity >0.8.12. most people still use 0.8.4, mostly because OZ uses that version in their templates. i think we should use the old abi encoding style for concatenating

Reply:
running `npx hardhat compile` leads to compile errors, likely because we're compiling against 0.8.4

Reply:
cool, i'll do that. i hadn't gotten as far as compile or tests since the lib was a weekend whim (outside of planning) i just wanted to make sure it tracked with where we wanted to take things before going too far. will clean it up now